plugins/teststeps/cmd/cmd.go

```diff
@@ -149,8 +153,11 @@ func (ts *Cmd) validateAndPopulate(params test.TestStepParameters) error {
 		ts.executable = p
 	}
 	ts.args = params.Get("args")
-	dir := params.GetOne("dir")
-	ts.dir = dir.String()
+	if !params.GetOne("dir").IsEmpty() {
+		ts.dir = params.GetOne("dir")
+	} else {
+		ts.dir = test.NewParam("")
+	}
```

Comment:
Isn't `GetOne` already returning a valid, empty one if unset? I think you don't need this.
```suggestion
      ts.dir = params.GetOne("dir")
```